.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.188.0/containers/cpp/.devcontainer/base.Dockerfile
+
+ARG VARIANT="buster"
+FROM mcr.microsoft.com/vscode/devcontainers/cpp:0-${VARIANT}
+
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends cmake \
+    ninja-build python3 clang-8 clang-format clang-tools \
```

Comment:
It doesn't have to be `clang-8`, it could be any clang 8 or later.

Reply:
Would you like me to change it to the latest version available?

Reply:
Or probably just leave as `clang`?

Reply:
Done, will commit in a second.

Reply:
We need clang 9 if people want to use clang-format and have things pass in CI (annoyingly, not clang-9-or-later, because the output isn't quite stable across releases).  I tend to use the latest release.  At least for the process sandboxing code, we should work with any recentish clang but need gcc 10 or newer because `__has_include` in older GCC is broken.